src/app/layout/qgslayoutlegendwidget.cpp

```diff
@@ -362,11 +429,23 @@ void QgsLayoutLegendWidget::mGroupSpaceSpinBox_valueChanged( double d )
   }
 }
 
+void QgsLayoutLegendWidget::spaceBelowGroupHeadingChanged( double space )
+{
+  if ( mLegend )
+  {
+    mLegend->beginCommand( tr( "Change Group Space" ), QgsLayoutItem::UndoLegendGroupSpace );
+    mLegend->rstyle( QgsLegendStyle::Group ).setMargin( QgsLegendStyle::Bottom, space );
+    mLegend->adjustBoxSize();
+    mLegend->update();
+    mLegend->endCommand();
+  }
+}
+
 void QgsLayoutLegendWidget::mLayerSpaceSpinBox_valueChanged( double d )
 {
   if ( mLegend )
   {
-    mLegend->beginCommand( tr( "Change Layer Space" ), QgsLayoutItem::UndoLegendLayerSpace );
+    mLegend->beginCommand( tr( "Change Subgroup Space" ), QgsLayoutItem::UndoLegendLayerSpace );
```

Comment:
Why changing layer to subgroup in the command description?

Reply:
We only ever refer to "subgroups" in the UI - "layer" is an internal term. (The background behind this decision predates me!)